### PR TITLE
Teet 1171 responsive login page

### PR DESCRIPTION
**Pull request by jalef**

### CODE REVIEW

File: app/frontend/src/cljs/teet/navigation/navigation_style.cljs [hunk 1]
```diff
@@ -123,34 +130,59 @@
...
-                                                  :padding "0 0.25rem"}))))
+                                                  :padding "0"}))))
 
+(defn open-account-navigation-style
+  []
+  (with-meta
+    {}
+    (responsivity-styles/desktop-only-meta
+      {:padding-left "1rem"}))
+  )
```
Comment (jkorpelin) [inline app/frontend/src/cljs/teet/navigation/navigation_style.cljs:156]: Fix indentation

File: app/frontend/src/cljs/teet/navigation/navigation_style.cljs [hunk 2]
```diff
@@ -123,34 +130,59 @@
 
 (defn language-select-style
   []
-  {:color theme-colors/blue})
+  ^{:pseudo {:focus {:min-width "3rem" :width :auto}}}
```
Comment (jkorpelin) [inline app/frontend/src/cljs/teet/navigation/navigation_style.cljs:133]: Indentation

File: app/frontend/src/cljs/teet/navigation/navigation_style.cljs [hunk 3]
```diff
@@ -53,24 +58,27 @@
...
+      :top 0
+      :bottom :auto
+      :background-color theme-colors/white
+      :color theme-colors/gray-dark
+      :height theme-spacing/appbar-height
+      :box-shadow "0px 2px 4px rgba(0, 0, 0, 0.36)"
+      :transition "all 0.2s ease-in-out"}
+    (responsivity-styles/mobile-only-meta
+      {:height theme-spacing/appbar-height-mobile})
+    ))
```
Comment (jkorpelin) [inline app/frontend/src/cljs/teet/navigation/navigation_style.cljs:72]: wrong line

File: app/frontend/src/cljs/teet/ui/buttons.cljs [hunk 4]
```diff
@@ -75,6 +75,11 @@
   (util/make-component Button {:variant        :contained
                                :disable-ripple true
                                :color          :primary}))
+(def large-button-primary
+  (util/make-component Button {:variant        :contained
```
Comment (jkorpelin) [inline app/frontend/src/cljs/teet/ui/buttons.cljs:79]: indentation

File: app/frontend/src/cljs/teet/search/search_view.cljs [hunk 5]
```diff
@@ -62,12 +96,24 @@
...
+                                        :margin-top "0.25rem"
+                                        :margin-bottom "0.25rem"
+                                        :margin-right "1rem"}}
+                         icon])
+                      [ListItemText
+                       {:disable-typography true
+                        :primary text}]]))
                  results)]
-               [CircularProgress])]])]))))
+               [CircularProgress { :size "20" :class (<class
```
Comment (jkorpelin) [inline app/frontend/src/cljs/teet/search/search_view.cljs:116]: indentation

**Review by jkorpelin — APPROVED** [review]
Other than the few indendation issues LGTM

File: app/frontend/src/cljs/teet/ui/buttons.cljs (hunk 4 above)
Comment (jalef) [reply app/frontend/src/cljs/teet/ui/buttons.cljs:79]: I will not fix this, because the indentation has the same format in the whole file

**Pull request merged**